### PR TITLE
Updated README and CHANGELOG for 2.2 release.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
Rename CHANGELOG to CHANGELOG.rst to show a nice page with linkable sections on GitHub.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -13,9 +13,9 @@ https://openslides.org/ for more information.
 Requirements
 ============
 
-OpenSlides runs everywhere where Python is running (for example on
-GNU/Linux, Mac or Windows (XP or newer)). On each client you need only a
-current version of a webbrowser.
+The OpenSlides server runs everywhere where Python is running (for example on
+GNU/Linux, Mac or Windows (XP or newer)). For OpenSlides clients a current
```
Comment (FinnStutzenstein) [inline README.rst]: `For the OpenSlides client`

File: README.rst [hunk 2]
```diff
@@ -60,15 +57,18 @@ Setup and activate the virtual environment::
 c. Install OpenSlides
 '''''''''''''''''''''
 
-To install OpenSlides just run::
+To install OpenSlides justn run::
```
Comment (FinnStutzenstein) [inline README.rst]: Typo

File: README.rst (hunk 1 above)
Comment (emanuelschuetze) [reply README.rst]: done

File: README.rst (hunk 2 above)
Comment (emanuelschuetze) [reply README.rst]: done

File: README.rst [hunk 3]
```diff
@@ -27,10 +26,15 @@ Installation
...
-Python (``python3-dev``).
+Make sure that you have installed `Python (>= 3.4) <https://www.python.org/>`_
+on your system.
+
+Additional you need the build-essential package, header files and a static
+library for Python and also the pyvenv-3 binary package for python3.
+
+E.g. run on Debian/Ubuntu::
+
+    $ sudo apt-get install build-essential python3-dev python3-venv``
```
Comment (normanjaeckel) [inline README.rst]: Remove ` at the end of the line.

**Comment by emanuelschuetze** [comment]
All done as discussed with @normanjaeckel

File: README.rst [hunk 4]
```diff
@@ -27,10 +26,15 @@ Installation
...
 '''''''''''''''''''''
 
-Make sure that you have installed `Python (>= 3.4)
-<https://www.python.org/>`_ on your system. You also need build-essential
-packages (``build-essential``) and header files and a static library for
-Python (``python3-dev``).
+Make sure that you have installed `Python (>= 3.4) <https://www.python.org/>`_
+on your system.
+
+Additional you need the build-essential package, header files and a static
```
Comment (normanjaeckel) [inline README.rst]:
```
Additional you need build-essential packages, header files and a static
```

File: README.rst (hunk 3 above)
Comment (emanuelschuetze) [reply README.rst]: done

File: README.rst (hunk 4 above)
Comment (emanuelschuetze) [reply README.rst]: done

**Pull request merged**